libcudacxx/include/cuda/std/detail/libcxx/include/__cccl/ptx_isa.h

```diff
@@ -31,47 +32,79 @@
  */
 
 // PTX ISA 8.3 is available from CUDA 12.3, driver r545
-#if   (defined(__CUDACC_VER_MAJOR__) && (__CUDACC_VER_MAJOR__ >= 12 && __CUDACC_VER_MINOR__ >= 3)) || (!defined(__CUDACC_VER_MAJOR__))
+#if   (defined(__CUDACC_VER_MAJOR__) && (__CUDACC_VER_MAJOR__ > 12)) || (!defined(__CUDACC_VER_MAJOR__))
+#  define __cccl_ptx_isa 830ULL
```

Comment:
We need a comment here to explain why we have 830ULL twice.. The difference between `> 12` and `>= 12` is too subtle for future contributors imho.